packages/edit-widgets/src/store/actions.js

```diff
@@ -101,6 +101,11 @@ export function* saveWidgetArea( widgetAreaId ) {
 		buildWidgetAreaPostId( widgetAreaId )
 	);
 
+	// Get all widgets from this area
+	const areaWidgets = Object.values( widgets ).filter(
```

Comment:
Is `Object.values` necessary? I think that the `getWidgets()` selector already returns an array.

Reply:
It's an object with `id` as key and the widget as value.